### DISCUSSION TITLE
PapersWithCode 值得参考

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-perf#15] [General]
Papers with Code 中收集了各种机器学习的内容（论文、代码、结果），为研究者们提供了迅速搜索论文与掌握行业领先研究的途径，一向受到「MLer」的欢迎。

https://mp.weixin.qq.com/s/HulhjMMQEnu42tFtcm7bqg

**Comment by bifenglin** [comment]
这个对于我们学习没什么问题，但是有人将这些问题抽象分类一下，形成结构化体系可能会更好些。
